config/system/ata.yaml

```diff
@@ -19,7 +22,10 @@ switch_setup:
 
 run_trigger:
   native_vlan: 122
-  max_hosts: 3
+  max_hosts: 10
+  block_time: 10
```

Comment:
I think this and scan_interval should have a _sec postfix to match existing default_timeout_sec long_dhcp_response_sec etc. Also could there be another name for scan_interval? The name is very close to monitor_scan_sec.